src/oc_erchef/apps/chef_index/c_src/fast_xs.c

```diff
@@ -0,0 +1,210 @@
+#include <assert.h>
```

Comment:
Is it an option to move the chef_reindex application into oc_erchef, or at least under chef-server/src/chef_reindex?